pacemaker.spec.in

```diff
@@ -399,6 +399,10 @@ sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
 
 make %{_smp_mflags} V=1 all
 
+%check
+# Prevent false positives in rpmlint
+./BasicSanity.sh -V pengine cli 2>&1 | sed s/[fF]ail/faiil/g
```

Comment:
I'm leaning against using the PE regression tests as a self-test. That's not really their intent, and it adds a large amount of time to rpmbuild unnecessarily. I think it would be better to define a "make check" target that runs maybe a single PE regression test and the cli tests, and call that here.

BTW we need to update the cli expected patterns